Phoenix/Server/Source/main.cpp

```diff
@@ -33,6 +38,7 @@ int main(int argc, char* argv[])
 	}*/
 
 	// ===== Begin shutdown =====
+	std::cout << "Begin Shutdown \n \n";
```

Comment:
This should probably use quartz logging functionality, instead of `cout`.

Reply:
This is just temp code to mark that things are happening, it will need to be replaced with something permanent in the future but we also still need it outputting to the terminal in the meantime.

Reply:
Cool, I did wonder if that was the case.